.travis.yml

```diff
@@ -126,7 +127,6 @@ before_install:
 install:
     - mkdir -p ~/.config/matplotlib
     - echo "backend:agg" > ~/.config/matplotlib/matplotlibrc
-    - ./scripts/travis/install-openblas.sh $OPENBLAS_ROOT
```

Comment:
The problem we are seeing though is that building openblas locally breaks something in numpy which is required by tensorflow. Maybe we can only build openblas in the torch build and leave tensorflow alone?